api/python/quilt3/workflows/config-1.schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://quiltdata.com/workflows/config/1",
+  "type": "object",
+  "required": [
+    "version",
+    "workflows"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "version": {
+      "const": 1
+    },
+    "is_workflow_required": {
+      "type": "boolean",
+      "description": "Whether package can be pushed with unspecified workflow.",
+      "default": false
+    },
+    "default_workflow": {
+      "type": "string",
+      "description": "The workflow that will be used if no workflow provided by user."
+    },
+    "workflows": {
+      "type": "object",
+      "minProperties": 1,
+      "propertyNames": {
+        "pattern": "^[A-Za-z_-][A-Za-z0-9_-]*$",
+        "maxLength": 64
+      },
```

Comment:
I don't have clear reasons for these constraints, I added them just in case we'd want to rework this somehow in future: like putting workflows in separate files or something like this.
Does that make sense?